lib/EVTutorial/ROOT

```diff
@@ -0,0 +1,15 @@
+(*
+ * Copyright 2020, The University of Melbourne (ABN 84 002 705 224)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ *)
+
+chapter Lib
+
+session EVTutorial = Lib +
+  options [document = pdf, document_output = "output"]
```

Comment:
Unless you really want the user to inspect the generated sources, I would usually not add an `output` option.

```suggestion
  options [document = pdf]
```